Shopify/dawn
# Vol pricing popover for Variant List

### PR Summary: 

Introduces a popover for variant list (to be added to cart later) where we can access qty rules and vol pricing

### Why are these changes introduced?

Designs: https://www.figma.com/file/ov2FHR3uvfVzKqbVV33IPz/Variant-list-on-PDP?type=design&node-id=3462-100096&t=Qc1uvy4PTYLha2lZ-0

Designs mobile: https://www.figma.com/file/ov2FHR3uvfVzKqbVV33IPz/Variant-list-on-PDP?type=design&node-id=3609-110752&t=rNYBtu51iV5cP7Yg-0


### Testing steps/scenarios
Login as a B2B to see it. Instructions [here](https://docs.google.com/document/d/1q3n39SzlKYGWWYaGQlY6ZK7cES3HUs6mxKAjBF2l-6U/edit)

- [ ] Test products with qty rules only
- [ ] Test products with qty rules + vol pricing (bo ivy black)
- [ ] Test products with vol pricing only (A/ Image sizing test – Different aspect ratios - Landscape first)
- [ ] Test products with no vol pricing nor qty rules
- [ ] Test mobile
- [ ] Popover to follow popover global settings and input to follow input global settings

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/140103712790/editor)

### Checklist
